meta-lmp-base/recipes-support/fioconfig/fioconfig/fioconfig.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Foundries.io configuration management daemon
+After=network.target
+ConditionPathExists=|/var/sota/sota.toml
+
+[Service]
+RestartSec=180
```

Comment:
You can probably set a smaller value here, as in reality this is a critical service.